boards/x86/qemu_x86/Kconfig.board

```diff
@@ -4,3 +4,4 @@ config BOARD_QEMU_X86
 	depends on SOC_IA32
 	select QEMU_TARGET
 	select HAS_DTS
+	select CPU_HAS_FPU if !X86_IAMCU
```

Comment:
is this typo? HAS_FPU instead of CPU_HAS_FPU.

Reply:
no, this is correct, the other one was a typo

Reply:
which is fixed now